app/src/main/java/net/squanchy/proximity/preconditions/OptInPreferencePersister.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.proximity.preconditions;
+
+import android.content.SharedPreferences;
+
+public class OptInPreferencePersister {
```

Comment:
Can we call this `ProximityOptInPersister` for consistency with the other similar classes?